cli/tests/integration_tests.rs

```diff
@@ -76,8 +77,9 @@ fn actual_empty_zcashd_sapling_commitment_tree() {
 
 #[test]
 fn mine_sapling_to_self() {
-    let (regtest_manager, _child_process_handler, client) = saplingcoinbasebacked_spendcapable();
-
+    let (regtest_manager, _child_process_handler, mut client_builder) =
+        saplingcoinbasebacked_spendcapable();
+    let client = client_builder.new_sameseed_client(0, false);
```

Comment:
Slight regression here in terms of DRY, this and at least the next test call this line immediately.

Reply:
Not a blocking concern